Gopkg.lock

```diff
@@ -0,0 +1,309 @@
+# This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
```

Comment:
This one is necessary by `go dep` to ensure we ahve the good versions when we install the dependencies